### ISSUE TITLE
Cutted "slow motion" shows wrong part of the clip in rendered video

### ISSUE THREAD

**Issue by nocxsville** [OpenShot/openshot-qt#3651]
**Describe the bug:**
Rendered video contains the wrong part of a cutted and slowed down clip.
In the preview everything is looking good. However, in the rendered video another part / the wrong part of the origional video is shown. This issue does not show up if the slow motion clip is at the beginning and the only clip in the tracks.

**Steps to reproduce the behavior:**
1. Add a video (several minutes) to the project
2. Drop the video to a track
3. Cut the first x minutes and last x minutes from the video (keep some seconds in the middle of the video)
4. Copy the cutted part and paste it after the first one (so you have the same clip twice in the track)
5. Slow down the speed of the second one: Time > Slow > Forward > 1/2 Speed
6. Render it

**Expected behavior:**
The right part of the clip should be in the rendered video

**System Details:**
- OpenShot Version 2.5.0 and 2.5.1
- Operating System / Distro: Windows 10

**Comment by nocxsville** [comment]
Seems to be a duplicate of https://github.com/OpenShot/openshot-qt/issues/3647

**Comment by Blitheness** [comment]
This just happened to me. Slowed down a clip, and the export did not match the preview. A little disappointed now as I feel like I've got to learn how to use an alternative program.

**Comment by khaines423** [comment]
I also just had this happen to one of my projects.  Please re-open.

**Comment by Blitheness** [comment]
When I had this problem the first slowed clip was fine, but subsequent clips rendered from the wrong position and inconsistent with the preview.

**Comment by Maherica** [comment]
I've resolved it saving a slowed clip as a separate file, but very complicated, especially if there are many clips. It would be very useful to fix it.

**Comment by JacksonRG** [comment]
@sm0kew0n Thanks for pointing this out! We'll take a look. Fixing this might help a few issues I've seen posted.

**Comment by schaden-fred** [comment]
I'm also experiencing this when using a video segment with the playback speed increased.  I can provide an example project and output file if needed.

**Comment by concolor22** [comment]
This bug is alive and well. Daily build, Oct 8, 2021

**Comment by IsmaelGO2021** [comment]
This bug still going on.... out 14th 2021....even cutting and paste the part to be slowed the bug persists  .... wondering when they will fix this because this app is so friendly to use....free.... hoping they fix it soon,

**Comment by IsmaelGO2021** [comment]
just in time: I'm using the lasted version for Windows 10 64bits

**Comment by DanPinGF** [comment]
Umm... if you're actually on Windows shouldn't you be expecting a crash when applying 'Time' to audio/video (#1197)? It's a long-term issue we've been seeing lately.

**Comment by ThomArmax** [comment]
I have the same issue with `2.6.0` on Ubuntu 20.04

**Comment by JacksonRG** [comment]
Just tried thee steps from the description with no problem (also on Ubuntu 20.04 but with the current development version)
@ThomArmax could you check if this issue is in the current daily build?
https://www.openshot.org/download/

**Comment by ThomArmax** [comment]
@JacksonRG I have just tried with the AppImage from the given link and still have the issue :
![image](https://user-images.githubusercontent.com/5956969/157826501-907f9494-6e91-4182-a2e7-8d13d03f7160.png)
But works with [OpenShot-v2.6.1-dev-daily-9246-f29ce408-2b8b05c7-x86_64.AppImage](https://github.com/OpenShot/openshot-qt/releases/download/daily/OpenShot-v2.6.1-dev-daily-9246-f29ce408-2b8b05c7-x86_64.AppImage)

**Comment by Homitu** [comment]
8/15/2022 update: I encountered this bug on my recent project.  Hence how I found myself here.  

What seemed to fix it for me was to export my video at 30fps (I tested at 720p and 1440p resolutions.)  Doing that caused the end product to match the preview video, which correctly displayed the stretched and slowed down clip.  

Exporting at different settings yielded different results.  In one instance, I encountered exactly what OP described.  The slowed down clip would start, then suddenly just cut to a completely different, earlier part of the video, which was NOT present at all on my timeline at that point.  Another instance just had the video freeze at that point.  

Hope that info helps in identifying the issue!
